reconcile/gitlab_permissions.py

```diff
@@ -77,9 +77,11 @@ def share_project_with_group_members(
 
 
 def share_project_with_group(gl: GitLabApi, repos: list[str], dry_run: bool) -> None:
+    # get repos not owned by app-sre
+    non_app_sre_projects = {repo for repo in repos if "/app-sre/" not in repo}
```

Comment:
can we get this via gitlab api instead of check url string?

Reply:
As discussed...Will be taken care in generic implementation...